.github/workflows/build-ci.yml

```diff
@@ -28,19 +28,18 @@ jobs:
                 laravel:
                     - "10.*"
                     - "11.*"
+                    - "12.*"
                 include:
                     - php: "8.1"
                       laravel: "10.*"
                       mongodb: "5.0"
                       mode: "low-deps"
                       os: "ubuntu-latest"
-                    - php: "8.4"
-                      laravel: "11.*"
-                      mongodb: "7.0"
-                      os: "ubuntu-latest"
                 exclude:
                     - php: "8.1"
                       laravel: "11.*"
+                    - php: "8.1"
+                      laravel: "12.*"
```

Comment:
Is this excluded because Laravel 11.x and 12.x require PHP 8.2+? If so, considering adding a comment above for future reference.

Reply:
PHP 8.1 is not supported since laravel 11.
https://github.com/laravel/framework/blob/11.x/composer.json#L18